### ISSUE TITLE
Failed Login for default Super Admin Account.

### ISSUE THREAD

**Issue by tandasima** [apiato/apiato#415]
## Expected Behavior

Expected to be able to login to the admin dashboard

## Actual Behavior

Getting Login Failed message.
```
[2018-04-27 19:53:57] local.ERROR: [ERROR] Status Code: 400 | Message: Login failed. | Errors:  | Code: 0  
```
## Steps to Reproduce the Problem

1. - Install Apiato using composer, version as of April, 26th 2018.
2. - composer install
3. - php artisan key:generate
4. - php artisan migrate
5. - php artisan db:Seed
6. - php artisan apiato:permissions:toRole admin
7. - php artisan passport:install
8. - vendor/bin/phpunit
9. - Go to admin.{yourdomain.here}/
Try to login using 
email: admin@admin.com
password: admin

## Specifications
 Running on Apache

  - Apiato Version: One from composer.

## Additional Details
I also noticed that in the database the super admin confirmed column is false. I do not know how to confirm a user. I overwrote it but was still unable to login. Also if I use an incorrect password, seems like it fails too with same 'Login Failed' messsage.

**Comment by johannesschobel** [comment]
Do you try to login with the API or WEB routes? For API you need to pass the `accept : application/json` headers..

Are you passing the client secrets properly?

**Comment by tandasima** [comment]
I am trying to login thru Web.

**Comment by tandasima** [comment]
@johannesschobel I was able to fix it. The password parameter in AuthorizationDefaultUsersSeeder_3.php is being Hashed hence causing it to be hashed again when CreateUserByCredentialsTask is called. This will cause the password stored in the database to be double hashed. It will fail on login since it will be reverse-hashed once only.

**FYI**
So it looks like the Version of Apiato installed using composer is different from the latest release tab here on Github. I have already found 2 bugs now, including this one, from the one installed using composer. Just putting it here fyi.

I think I will download one from the release tab going on.
